Homework7/Item.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Homework7
+{
+    public abstract class Item 
+    {
+        public string? modelName;
```

Comment:
As these fields are being used only in derived classes it worse to make them protected

Reply:
fixed